src/scripts.js

```diff
@@ -203,5 +212,22 @@ function filterInputs(letters,ingredientsData,currentUserProperty){
   })
 }
 
+hamburgerIcon.addEventListener('click', ()=>{
```

Comment:
The `hamburgerIcon` and `closeIcon` have a lot going on as far as event handlers go.  Maybe we can check out a `.toggle()` method (pretty sure one exists somewhere on MDN) since there is a lot of `add()` and `remove()` going on  so there are opportunities to DRY the code.